jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/annotations/CloudFormationLintAnnotator.kt

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.cloudformation.annotations
+
+import com.intellij.lang.annotation.AnnotationHolder
+import com.intellij.lang.annotation.ExternalAnnotator
+import com.intellij.openapi.fileEditor.FileDocumentManager
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiFile
+
+/**
+ *
+ * Process the files in a language by an external annotation tool ("linter").
```

Comment:
Do we feel like all these comments are required?

Reply:
They are by no means required but they did help me a lot to better structure the code